src/app/modules/OpenshiftStreams/components/StreamsTable/StreamsTable.tsx

```diff
@@ -255,6 +255,7 @@ export const StreamsTable: FunctionComponent<StreamsTableProps> = ({
       {
         title: t("view_details"),
         id: "view-instanceDrawerInstance",
+        disabled: false,
```

Comment:
we don't need to add disabled property. You should remove additionalProps and tooltipProps where we need to show kebab options for all users.